EssayAnalyzer.Api/Services/Foundation/Users/UserService.Exception.cs

```diff
@@ -49,11 +49,27 @@ private async ValueTask<User> TryCatch(ReturnUserFunction returnUserFunction)
         }
         catch (DbUpdateException dbUpdateException)
         {
-            var failedUserStorageException = 
+            var failedUserStorageException =
                 new FailedUserStorageException(dbUpdateException);
 
```

Comment:
Make it single line

Reply:
It is long named exception if I use it in one line, it will be more than method name.